src/Stores/UIStore.ts

```diff
@@ -0,0 +1,42 @@
+// uiStore.ts
+import { observable, action, computed , makeObservable} from 'mobx';
+import memoize from 'lodash.memoize';
+class UiStore {
+  
+   _isDrawerOpen: boolean = false;
+   _selectedLanguage : string = 'English';
+   constructor()
+   {
+    makeObservable(this,{
```

Comment:
no need at all. let it be last comment about this part please

Reply:
Done